lib/gamnit/flat.nit

```diff
@@ -1077,7 +1077,7 @@ end
 #
 # array.add "E"
 # assert array.to_s == "[A,B,c,D,E]"
-# assert array.capacity == 5
+# assert array.capacity == 6
```

Comment:
I'm not fan of testing the capacity. The only contract that make sense here is `assert array.capacity >= 5`